### ISSUE TITLE
IFTTT event not triggered

### ISSUE THREAD

**Issue by AJolly** [steve-vincent/DictateIFTTT#1]
Trying to diagnose this - 
Seems like the IFTTT event works intermittently.
Dictation will work successfully, it will say the message is sent, but don't receive anything on the IFTTT end.  Any idea?

**Comment by decentropy** [comment]
Intermittent? not sure what that would be.

**Comment by AJolly** [comment]
Yeah, really not sure.  No error messages are displayed on the Pebble app side.  IFTTT says the event never ran.  The voice to text part works correctly, I'm not sure if it's a lack of error correction if the phone fails to make a connection to IFTTT or what.
